tasks/keys/setup-keys.yml

```diff
@@ -4,12 +4,12 @@
   when: ansible_os_family == "Alpine"
   tags: nginx_apkkey
 
-- name: "(Setup: Keys) Debian"
+- name: "(Setup: Keys) Debian/Ubuntu"
   include_tasks: "{{ role_path }}/tasks/keys/apt-key.yml"
   when: ansible_os_family == "Debian"
```

Comment:
if you want to add 'Ubuntu' support in addition to 'Debian' - it is needed to add a contain that covers 'Ubuntu'

``` when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu' ```

Reply:
Ubuntu's `ansible_os_family` is Debian.